### PR TITLE
Add cypress parameters for data source

### PR DESCRIPTION
**Pull request by DhananjayPurohit**
Signed-off-by: DhananjayPurohit <dhananjaypurohit7@gmail.com>

## Proposed changes

Add cypress parameters for data source creation, configuration, and deletion

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by ishangupta-ds — APPROVED** [review]

File: litmus-portal/frontend/src/pages/ConfigureDataSources/index.tsx [hunk 1]
```diff
@@ -254,6 +254,7 @@ const DataSourceConfigurePage: React.FC<DataSourceConfigurePageProps> = ({
               onClick={() =>
                 page === 2 && !disabled ? setMutate(true) : setPage(2)
               }
+              data-cy="dataSourceControl"
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/pages/ConfigureDataSources/index.tsx:257]: It should be `dataSourceControlButton`

**Review by Jonsy13 — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/views/Observability/MonitoringDashboards/Table/TableData.tsx [hunk 2]
```diff
@@ -321,7 +321,7 @@ const TableData: React.FC<TableDataProps> = ({ data, alertStateHandler }) => {
                 className={classes.btnImg}
               />
               <Typography
-                data-cy="configureDashboard"
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/views/Observability/MonitoringDashboards/Table/TableData.tsx:324]: No need to change here, this file is for dashboard.

Comment (DhananjayPurohit) [reply litmus-portal/frontend/src/views/Observability/MonitoringDashboards/Table/TableData.tsx]: Undo that. Thanks.

File: litmus-portal/frontend/src/views/Observability/MonitoringDashboards/Table/TableData.tsx [hunk 3]
```diff
@@ -367,7 +367,7 @@ const TableData: React.FC<TableDataProps> = ({ data, alertStateHandler }) => {
                 className={classes.btnImg}
               />
               <Typography
-                data-cy="deleteDashboard"
+                data-cy="deleteDatasource"
```
Comment (Jonsy13) [inline litmus-portal/frontend/src/views/Observability/MonitoringDashboards/Table/TableData.tsx:370]: Please undo this as well.

Comment (DhananjayPurohit) [reply litmus-portal/frontend/src/views/Observability/MonitoringDashboards/Table/TableData.tsx]: Undo that. Thanks again.

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**